lib/krane/deploy_task.rb

```diff
@@ -144,7 +144,7 @@ def run!(verify_result: true, prune: true)
       @logger.phase_heading("Initializing deploy")
       validate_configuration(prune: prune)
       resources = discover_resources
-      validate_resources(resources)
+      validate_resources(resources, prune)
```

Comment:
`prune` probably not really necessary for just dry-running...

Reply:
we could always remove this here and have `validate_dry_run` pass false for the prune flag automatically as an idea

Reply:
After thinking about it, I don't think we need the prune flag at all, since no human will see the output. wdyt @dturn ?